Dockerfile

```diff
@@ -53,3 +52,35 @@ COPY extend-environment.sh /etc/profile.d/extend-environment.sh
 RUN pip install --no-cache-dir -r requirements/local.txt
 
 CMD ["python", "./cfgov/manage.py", "runserver", "0.0.0.0:8000"]
+
+FROM cfgov-develop as cfgov-build
+
+# add node and yarn repos
+RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash -
+RUN curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo
+
+RUN yum install -y nodejs yarn  && \
+    yum clean all && rm -rf /var/cache/yum
+
+WORKDIR /src/cfgov-refresh
+
+COPY scripts scripts
```

Comment:
Perhaps sort these alphabetically?

Reply:
+1 to alphabetically, but I'd also think it's worthwhile to include these all in one `COPY` (space-separated), then they'll only be a single layer

Reply:
Yeah, I only recently learned that was possible-- I'll make that change

Reply:
I was kind of defeated in my attempt to put them all on one line-- everything I tried to do with directories, ended up with the *contents* of the directories being dropped into /src/cfgov-refresh. Any ideas are welcome, but in the meantime I just copied the directories separately from the single files.